### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_MenuBar_ScreenReader: Narrator in not announcing the Expand/Collapse state for Menuitems(Edit,View,Insert,Cell...etc).

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3985]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Operating System:Windows10

**ReproSteps:**
1. Login with the appropriate credentials to azure notebook account.    
2. Navigate to any of the file names present in the samples-->>clickin the file name-->Welcome page opens with menubar.
3. Start Narrator.
4. Navigate till menubar and Verify Narrator is not announcing Expand/Collapse state for any of the menuitems(Edit,View,Insert,Cell...etc).

**Actual:**
Narrator in not announcing the Expand/Collapse state for Menuitems(File,Edit,View,Insert,Cell...etc).

**Note:**
Same issue is observed in **NVDA** also.

Same issue is observed for "RunningTab" page for "Terminals" and "NoteBooks" Dropdowns.
1. RunningTab--> Navigate to run button & hit enter --> Navigate to running tab & hit enter.

**Expected:**
Narrator should announce the Expand/Collapse state for all Menuitems(File,Edit,View,Insert,Cell...etc).
For Example: When "File" menu is Expanded Narrator should announce "File"Menu Expanded. And When "File" menu is Collapsed Narrator should announce "File"Menu Collapsed.

**UserImpact:**
If ScreenReader not announcing Expand/Collapse state then Screenreader users may not be able to know control is expanding or not.

**Attachment:**
(https://github.com/jupyter/notebook/files/2400358/A11y_AzureNotebooks_Jupyter_MenuBar_ScreenReader.pptx)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
